rasp/engines/base.py

```diff
@@ -67,7 +66,7 @@ def as_func(cls, method_name):
         try:
             return getattr(TmpEngine, method_name)
         except AttributeError:
-            raise NotImplementedError('Class {} does not implement {}'.format(class_name, func_name))
+            raise NotImplementedError('Class {} does not implement {}'.format(class_name, method_name))
```

Comment:
Already fixed this [here](https://github.com/anidata/rasp/pull/26/files#diff-3387bc458c72e04ddfef789b19a5bcafR71).  Make sure to rebase/merge your branch on the current develop.